templates/includes/sidebar.html

```diff
@@ -2,3 +1,18 @@
+<ul style="list-style: none">
+  {% if user.is_authenticated %}
+    <p>Hello,
+      <strong><a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}!</a></strong>
+    </p>
+    <form action="{% url 'logout' %}" method="post">
+      {% csrf_token %}
+      <button type="submit" class="btn btn-link">
+        Logout
+      </button>
+    </form>
+  {% else %}
+    <a href="{% url 'login' %}">Login</a>
```

Comment:
The login link is correctly implemented. Ensure that the `login` URL is correctly configured in your authentication URLs.